doc/application/application.rst

```diff
@@ -935,155 +935,121 @@ The example below shows a comment line and an override setting
 
-#. Generate a Make build system, and use it to run ``ninja
-   menuconfig`` as follows.
+#. Create a build directory :file:`<home>/app/build` inside your application
+   directory and generate build files inside it with CMake, as follows:
 
-   a) Using CMake, create a build directory (:file:`<home>/app/build`) from
-      your application directory (:file:`<home>/app`).
+   .. code-block:: bash
 
-      For example, on a shell or command prompt:
-
-      .. code-block:: bash
+      # On Linux/macOS
+      cd ~/app
+      # On Windows
+      cd %userprofile%\app
 
-         # On Linux/macOS
-         cd ~/app
-         # On Windows
-         cd %userprofile%\app
+      mkdir build && cd build
+      cmake -GNinja ..
 
-         mkdir build && cd build
-         cmake -GNinja ..
+#. Run the following command from the build directory (:file:`<home>/app/build`)
+   to start the configuration interface:
 
-   b) Run ``ninja menuconfig`` from the build directory
-      (:file:`<home>/app/build`).
+   .. code-block:: bash
 
-      Continuing the above Unix shell example:
+       ninja menuconfig
 
-      .. code-block:: bash
+   The configuration interface is shown below:
 
-          ninja menuconfig
+   .. image:: figures/app_kernel_conf_1.png
+        :align: center
+        :alt: Main Configuration Menu
 
-      A question-based menu opens that allows you to set individual
-      configuration options.
+#. Set kernel configuration values as follows:
 
-      .. image:: figures/app_kernel_conf_1.png
-           :width: 600px
-           :align: center
-           :alt: Main Configuration Menu
+   * Use the arrow keys to navigate the menu.
 
-#. Set kernel configuration values using the following
-   key commands:
+     .. note::
 
-   * Use the arrow keys to navigate within any menu or list.
+        Common `Vim <https://www.vim.org>`_ key bindings are supported as well.
 
-   * Press :kbd:`Enter` to enter submenus and choices, which appear with
-     ``--->`` next to them.
+   * Press :kbd:`Enter` or :kbd:`Space` to enter submenus and choices, which
+     appear with ``--->`` next to them. Press :kbd:`ESC` to return to the
+     parent menu.
 
    * Press :kbd:`Space` to toggle or configure a symbol value. Boolean
      configuration symbols are shown with :guilabel:`[ ]` brackets, while
      numeric and string-valued configuration symbols are shown with
      :guilabel:`( )` brackets.
 
-   * Press :kbd:`Tab` to navigate the command menu at the bottom of the
-     display.
-
-#. For information about any symbol, select the symbol and type :kbd:`?`.
+     .. note::
 
-   Press :kbd:`Enter` to return to the menu.
+        You can also press :kbd:`Y` or :kbd:`N` to set a boolean configuration
+        symbol to the corresponding value.
 
-#. Press :kbd:`/` to bring up a search menu to look for a particular option.
+#. To view information about a symbol, select it and press :kbd:`?`.
```

Comment:
The previous key explanations were "active", so let's make this one as well:

    #. Press :kbd:`?` to view information about the selected symbol.

Reply:
Hmm... that makes it sound like something the user must do, since it's currently "step 4".

Think that's the real problem. I'll see if I can reorganize it.